hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/wal/AbstractTestFSWAL.java

```diff
@@ -186,12 +193,10 @@ protected void addEdits(WAL log, RegionInfo hri, TableDescriptor htd, int times,
 
   /**
    * helper method to simulate region flush for a WAL.
-   * @param wal
-   * @param regionEncodedName
    */
   protected void flushRegion(WAL wal, byte[] regionEncodedName, Set<byte[]> flushedFamilyNames) {
     wal.startCacheFlush(regionEncodedName, flushedFamilyNames);
-    wal.completeCacheFlush(regionEncodedName);
+    wal.completeCacheFlush(regionEncodedName, HConstants.NO_SEQNUM);
```

Comment:
Should completeCacheFlush check for this?

Reply:
What do you mean sir?

Reply:
This is just for test, where we want to simulate a flush, so it is fine.

Reply:
Makes sense.